tests/lint.sh

```diff
@@ -1,14 +1,26 @@
 #!/usr/bin/env sh
 set -ve
 
-ansible-playbook --syntax-check "cinch/site.yml" -i inventory/sample/hosts
-ansible-playbook --syntax-check "cinch/teardown.yml" -i inventory/sample/hosts
-
-find . -name '*.sh' -not -name 'jswarm.sh' -print0 | \
-	xargs -0 shellcheck -e 1090,1091,2093
-find . -name 'jswarm.sh' -print0 | xargs -0 shellcheck -e 1090,1091,2093,2086
-
-SOURCES=$(find . -name '*.py')
-for src in ${SOURCES}; do
-    flake8 "${src}"
-done
+###############################################################################
+# ANSIBLE LINT
+###############################################################################
+find cinch -maxdepth 1 -name '*.yml' -print0 |
+	xargs -0 -L 1 ansible-playbook \
+	             --syntax-check \
+	             -i inventory/sample/hosts
+find cinch -maxdepth 1 -name '*.yml' -print0 |
+	xargs -0 -L 1 ansible-lint
+###############################################################################
+# SHELL LINT
+###############################################################################
+find . -name '*.sh' -not -name 'jswarm.sh' -print0 |
```

Comment:
D'oh. I just added that commit to the Ansible 2.3 warnings branch. But this jswarm.sh exception has existed here since the script was created. I just changed the way that we're using find and other loops. The comment on this file will get merged in when the 2.3 changes get merged.